tests/test_5_analyze.py

```diff
@@ -18,22 +18,20 @@ def setup_and_cleanup():
     if not os.path.exists(test_bitcoin_dir):
         os.makedirs(test_bitcoin_dir)
     # create files that would be the output of mapping
-    csv_per_timeframes = {
-        '2010': 'Entity Group,Entity,Resources',
-        '2018': 'Entity Group,Entity,Resources\n'
-                '1AM2f...9pJUx/3G7y1...gPPWb,1AM2f...9pJUx/3G7y1...gPPWb,4\n'
-                'BTC.TOP,BTC.TOP,2\n'
-                'GBMiners,GBMiners,2\n'
-                'Unknown,1AM2fYfpY3ZeMeCKXmN66haoWxvB89pJUx,1',
-        '2018-02': 'Entity Group,Entity,Resources\n'
-                   '1AM2f...9pJUx/3G7y1...gPPWb,1AM2f...9pJUx/3G7y1...gPPWb,4\n'
-                   'BTC.TOP,BTC.TOP,2\n'
-                   'GBMiners,GBMiners,2',
-        '2018-03': 'Entity Group,Entity,Resources\n'
-                   'Unknown,1AM2fYfpY3ZeMeCKXmN66haoWxvB89pJUx,1'
-    }
+    csv_per_timeframes = {'2018': 'Entity Group,Entity,Resources\n'
+                                  '1AM2f...9pJUx/3G7y1...gPPWb,1AM2f...9pJUx/3G7y1...gPPWb,4\n'
+                                  'BTC.TOP,BTC.TOP,2\n'
+                                  'GBMiners,GBMiners,2\n'
+                                  'Unknown,1AM2fYfpY3ZeMeCKXmN66haoWxvB89pJUx,1',
+                          '2018-02': 'Entity Group,Entity,Resources\n'
+                                     '1AM2f...9pJUx/3G7y1...gPPWb,1AM2f...9pJUx/3G7y1...gPPWb,4\n'
+                                     'BTC.TOP,BTC.TOP,2\n'
+                                     'GBMiners,GBMiners,2', '2018-03': 'Entity Group,Entity,Resources\n'
```

Comment:
missing newline here

Reply:
sorry I don't understand why we need a newline here, could you elaborate please?

Reply:
ah right, I didn't see there was a different date there, nice catch - fixed now